pages/admin/add-a-trust.js

```diff
@@ -103,6 +117,10 @@ const AddATrust = () => {
       setPasswordConfirmationError(onSubmitErrors);
     }
 
+    if (videoProvider.length === 0) {
```

Comment:
Could we move this to after checking if there is an error in the name?

The reason being is the `Select a video provider` error message ends up in the bottom of the error summary. The error messages in the error summary should follow the order of the inputs on the page. It shucks that it's so tied to the presentation of it but not sure how to do it without a ton of refactoring. :/

![image](https://user-images.githubusercontent.com/42817036/85266642-0c0f0500-b46c-11ea-99aa-f38f0febd006.png)